src/m365/spo/commands/eventreceiver/eventreceiver-remove.ts

```diff
@@ -0,0 +1,221 @@
+import { Logger } from '../../../../cli/Logger';
+import Command from '../../../../Command';
+import { Cli, CommandOutput } from '../../../../cli/Cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import { formatting } from '../../../../utils/formatting';
+import { urlUtil } from '../../../../utils/urlUtil';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import { Options as SpoEventReceiverGetOptions } from './eventreceiver-get';
+import commands from '../../commands';
+import request from '../../../../request';
+
+const getCommand: Command = require('./eventreceiver-get');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  listUrl?: string;
+  scope?: string;
+  id?: string;
+  name?: string;
+  confirm?: boolean;
+}
+
+class SpoEventreceiverRemoveCommand extends SpoCommand {
+  public get name(): string {
+    return commands.EVENTRECEIVER_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Gets a specific event receiver attached to the web, site or list (if any of the list options are filled in) by receiver name of id and removes this one';
```

Comment:
Let us align the documentation with the new description

Reply:
I created an interface "EventReceiver". Also updated the eventreceiver-list and eventreceiver-get to use this interface.